service/routes.py

```diff
@@ -27,69 +27,61 @@
 from service.common import status  # HTTP Status Codes
 
 
+
+
 ######################################################################
-# GET INDEX
+# UPDATE AN EXISTING Payment
 ######################################################################
-@app.route("/")
-def index():
```

Comment:
it seems that you removed some code that should belong in master branch during your merge attempt

Reply:
fixed